src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -38,20 +38,143 @@ import pixieServiceOtelMap from 'images/pixie_screenshot-full_service-otel-map.w
+  />
+
+</TechTileGrid>
+
+            </td>
+        </tr>
+        <tr>
+            <td>
+ Manual EKS Fargate install
+            </td>
+            <td>
+When monitoring Kubernetes workloads on EKS Fargate. This integration automatically injects a sidecar containing the infrastructure agent and the nri-kubernetes integration in each pod that needs to be monitored.
+            </td>
+            <td>
+<TechTileGrid>
+  <TechTile
+    name="EKS Fargate"
+    icon="logo-aws"
+    to="/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/install-fargate-integration/"
+  />
+
+</TechTileGrid>
+
+            </td>
+        </tr>
+<tr>
+<td>New Relic operator</td>  
+<td>Kubernetes operators help manage these complex applications by abstracting those Kubernetes resources into a set of custom configurations, or custom resources.
+
+ This reduces the burden on you as a user, as you would only need to interface with the custom resources to manage the application. You can then rely on the operator to deploy, upgrade, and manage the application for you.
+
+</td>
+<td>
+<TechTileGrid>
+  <TechTile
+    name="New Relic operator"
+    icon="logo-newrelic"
+    to="/docs/kubernetes-pixie/kubernetes-integration/installation/install-with-operator/"
+  />
+
+</TechTileGrid>
+
+Follow the instructions in the operator doc and refer back to this doc as needed.</td>
+</tr>
+    </tbody>
+</table>
+
+The remainder of this doc will cover the guided install process.
+
+## Guided install [#before-start]
+
+Take a look at the following to make sure you're ready:
+
+* If custom manifests have been used instead of Helm, you will need to first remove the old installation using `kubectl delete -f previous-manifest-file.yml`, and then proceed through the guided installer again. This will generate an updated set of manifests that can be deployed using `kubectl apply -f manifest-file.yml`.
+* Make sure you're using the supported Kubernetes versions and make sure to check out the preliminary notes for your managed services or platforms on our [compatibility and requirements page](/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
+* Make sure you have New Relic <InlinePopover type="licenseKey" />. You can set up an account that's free&mdash;no credit card required.
+* Make sure the newrelic dockerhub (`https://hub.docker.com/u/newrelic`) and Google's registry (`registry.k8s.io`) domains are added to your allow list. This is where the installation will pull container images from. Note, you may need to follow the [commands](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/#how-can-i-check-if-i-am-impacted) to identify the additional Google registry domains to be added to your white list, because `registry.k8s.io` is typically redirected to your local registry domain (e.g., `asia-northeast1-docker.pkg.dev`) based on your region.
```

Comment:
is "newrelic dockerhub" the proper formatting for this? I'm not actually sure myself so I wanted to ask